fe/fe-core/src/main/java/com/starrocks/analysis/DescribeStmt.java

```diff
@@ -164,8 +164,8 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
                 ErrorReport.reportAnalysisException(ErrorCode.ERR_BAD_TABLE_ERROR, dbTableName.getTbl());
             }
 
-            if (table.getType() == TableType.HIVE) {
-                // Reuse the logic of `desc <table_name>` because hive external table doesn't support view.
+            if (table.getType() == TableType.HIVE || table.getType() == TableType.HUDI) {
```

Comment:
So in new framework, there will be mthod like `IsDescribeAllTablesSupported`?

Reply:
yes, it should be under consideration too @caneGuy